charts/karpenter/templates/clusterrole-core.yaml

```diff
@@ -30,7 +30,7 @@ metadata:
 rules:
   # Read
   - apiGroups: ["karpenter.sh"]
-    resources: ["provisioners", "provisioners/status"]
+    resources: ["provisioners", "provisioners/status", "machines", "machines/status"]
```

Comment:
Adding rules without having the resource is supported by K8s. This doesn't error.